gradle/libs.versions.toml

```
@@ -1,32 +1,28 @@
 [versions]
-loom = "1.3.+"
-loom_vineflower = "1.11.+"
+loom = "1.7.+"
 
 minotaur = "2.7.+"
 cursegradle = "2.+"
 github_release = "2.+"
 machete = "2.+"
 
-minecraft = "1.20.2"
-quilt_mappings = "2"
-fabric_loader = "0.14.22"
-fabric_api = "0.89.1+1.20.2"
-yacl = "3.2.1+1.20.2"
-mod_menu = "8.0.0"
-mixin_extras = "0.2.0-rc.4"
+minecraft = "1.21"
+quilt_mappings = "1"
```

```suggestion
quilt_mappings = "5"
```
Quilt mappings for MC 1.21 are at v5 (see https://quiltmc.org/en/usage/latest-versions/)

---

Updated, when I first ported this on 1.21 release day Quilt didn't even have 1.21 mappings out so it used the rc lmao. On a side note Adaptive Tooltips should probably switch away from QM and use either Yarn or Mojmap given the development situation at Quilt.